src/components/Home/Home.test.js

```diff
@@ -12,13 +15,63 @@ const fakeStore = {
   status: {},
 };
 
+const mockMovies = [
+  { id: 'foo', poster_path: 'path' },
+  { id: 'bar', poster_path: 'another_path' },
+];
+
 describe('Home', () => {
-  test('should render a welcome message with the user name', () => {
-    const { getByText } = renderWithProviders(<Home />, {
-      initialState: fakeStore,
+  test('should render a movies container', async () => {
+    useFetchMovies.mockReturnValue({
+      movies: mockMovies,
+      fetchMore: jest.fn(),
+      isFetching: false,
+      error: null,
     });
 
-    expect(getByText('Home')).toBeTruthy();
-    expect(getByText('Welcome John')).toBeTruthy();
+    const { getByTestId } = renderWithProviders(<Home />, {
+      initialState: fakeStore,
+    });
+    expect(getByTestId('moviesContainer').props).toMatchInlineSnapshot(`
+      Object {
+        "children": Array [
+          <Center
+            style={Object {}}
+          >
+            <Image
+              source={
+                Object {
+                  "testUri": "../../../src/assets/ic_logo/ic_logo.png",
+                }
+              }
+              style={
+                Object {
+                  "marginTop": 50,
+                }
+              }
+            />
+          </Center>,
+          false,
+        ],
+        "style": Object {
+          "backgroundColor": "black",
+        },
+        "testID": "moviesContainer",
+      }
+    `);
   });
+
+  // todo: fix this test, the error element isn't being found in the dom.
```

Comment:
Maybe you have to add some delay to wait for the library to render it? Just throwing a suggestion 🤷‍♂️

Reply:
Yes, that's a good suggestion, but actually I tried that with a setTimeout and with waitFor (which is just for that) and neither of those worked...